.github/workflows/run-tests.yml

```diff
@@ -10,9 +10,9 @@ jobs:
         runs-on: ${{ matrix.os }}
         strategy:
             matrix:
-                php: [7.4, 7.3, 7.2]
+                php: [7.4]
                 dependency-version: [prefer-lowest, prefer-stable]
-                os: [ubuntu-latest, windows-latest]
```

Comment:
would like to keep windows tests, if not even add macos. But at least windows sometimes behaves different than linux and this is a super basic package it should be battle-tested - also on windows.
We can discuss this again if tests are failing on windows without any reason/way to fix.^^

Reply:
Fair enough!

Reply:
https://github.com/spatie/enum/pull/56/commits/c42de09faa94f814cf2a84ef111ac7747b997292